### PR TITLE
new product overview page

### PR DESCRIPTION
**Pull request by sean1588**
New product overview page

### CODE REVIEW

File: themes/default/layouts/page/product.html [hunk 1]
```diff
@@ -111,46 +122,136 @@ <h5>{{ $feature.title }}</h5>
...
+                    <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                        {{ partial "color-icon.html" (dict "icon" "security" "icon_color" "purple") }}
+                        <h5>Integration testing</h5>
+                        <p class="key-feature-description">Run external black-box tests against ephemeral, running infrastructure.</p>
+                    </div>
+                    <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                        {{ partial "color-icon.html" (dict "icon" "monitor" "icon_color" "purple") }}
+                        <h5>Policy as Code testing</h5>
+                        <p class="key-feature-description">Create and run resource-level assertions using policies written in general-purpose languages.</p>
+                    </div>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:189]: missing the continous delivery section right after this section

File: themes/default/layouts/page/product.html [hunk 2]
```diff
@@ -111,46 +122,136 @@ <h5>{{ $feature.title }}</h5>
...
+    </section>
+
+    <section class="mt-12">
+        <div class="container mx-auto flex flex-col lg:flex-row">
+            <div class="w-full lg:w-1/2 pl-3 pr-16">
+                <h5>Manage your infrastructure as code through your existing Git and code review process</h5>
+            </div>
+
+            <div class="product-carousel-items">
+                <img src="/images/product/code-review.png" alt="Code review" />
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:158]: can you add the caption beneath this screenshot?

> GitHub is shown above. Pulumi integrates with GitHub repositories and GitHub actions.

File: themes/default/layouts/page/product.html [hunk 3]
```diff
@@ -41,19 +35,19 @@ <h2>{{ .Params.overview.title }}</h2>
 
                 <ul id="product-carousel-headers" class="product-carousel-headers">
                     <li class="" data-product-carousel-type="build">
-                        <p class="my-0 text-black"><a href="#build" class="text-black">Build</a></p>
+                        <p class="my-0 text-black"><a href="#build" class="text-black">Languages</a></p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:38]: for this whole section, could we just use what's currently on the /product/ page? it should be the same graphics that's currently on /product/ where it says Build/Deploy/Manage.

File: themes/default/content/product/_index.md [hunk 4]
```diff
@@ -12,80 +12,92 @@ subheading: |
...
+    title: Universal infrastructure as code for engineers 
     description: |
-        By using the industry's best languages, you can skip learning proprietary, limited domain-specific
-        languages, and enable your team to work more productively and better together.
+        Ship infrastructure at faster velocity compared to traditional tools because it supports the most popular programming languages and software engineering practices. 
 
 key_features:
     title: Key features
     items:
         - description: Build [Infrastructure as Code](/what-is/what-is-infrastructure-as-code/) in familiar languages
```
Comment (gthuang) [inline themes/default/content/product/_index.md:22]: this title should not be here. you can delete it.

File: themes/default/layouts/page/product.html (hunk 3 above)
Comment (sean1588) [reply themes/default/layouts/page/product.html]: sure. i was trying to match what was in the design mock ups. i will revert this section back to how the page currently is.

File: themes/default/content/product/_index.md (hunk 4 above)
Comment (sean1588) [reply themes/default/content/product/_index.md:24]: yeah, i will likely need to revisit the structure of this when i go back through and clean this up. it currently is not being rendered or used though. so you shouldnt be seeing it on the page.

Comment (sean1588) [reply themes/default/content/product/_index.md:24]: ahh nevermind. I see what you are saying. i thought you were referring to the title property.

File: themes/default/content/product/_index.md [hunk 5]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
+              - title:
                 icon: rocketship
                 icon_color: salmon
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
+                    Build faster by reusing Pulumi libraries and CDK constructs, and create your own libraries as multi-language Pulumi Packages.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:50]: link CDK constructs to: https://www.pulumi.com/blog/aws-cdk-on-pulumi/
link multi-language Pulumi Packages to https://www.pulumi.com/product/packages/
link Pulumi libraries: https://www.pulumi.com/docs/intro/concepts/resources/components/

File: themes/default/content/product/_index.md [hunk 6]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
 
-              - title: Deployments as Code
+              - title:
                 icon: gear
                 icon_color: salmon
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
+                    Develop modern cloud applications faster while using well-architected best practices with the Crosswalk for AWS library.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:56]: link Crosswalk for AWS to https://www.pulumi.com/docs/guides/crosswalk/aws/

File: themes/default/content/product/_index.md [hunk 7]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
 
-              - title: Preview and test changes
+              - title:
                 icon: eye
                 icon_color: salmon
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Convert your existing infrastructure code to Pulumi (TF, ARM, CFN, K8s, CRD, Helm) and import existing resources into Pulumi code.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:62]: link "convert" to https://www.pulumi.com/docs/converters/

and "import" to https://www.pulumi.com/docs/guides/adopting/import/

File: themes/default/content/product/_index.md [hunk 8]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
+     
+        - description: Deploy to over 100 cloud and SaaS providers
+          features:
+              - title: 
+                icon: shield
+                icon_color: purple
+                description: |
+                    Native providers for AWS, Azure, Google Cloud, and Kubernetes have 100% API coverage and same-day updates for new features.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:70]: link Azure to https://www.pulumi.com/blog/pulumiup-native-providers/
link AWS to https://www.pulumi.com/blog/announcing-aws-native/
link Kubernetes to https://www.pulumi.com/registry/packages/kubernetes/
link google Cloud to https://www.pulumi.com/blog/pulumiup-google-native-provider/

File: themes/default/content/product/_index.md [hunk 9]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
+
+              - title: 
+                icon: security
+                icon_color: purple
+                description: |
+                    Use a single workflow to manage cloud native infrastructure like Kubernetes, Helm, CRDs, GitOps, and more.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:76]: link kubernetes to https://www.pulumi.com/registry/packages/kubernetes/
link helm to https://www.pulumi.com/blog/helm-release-resource-for-kubernetes-generally-available/
link GitOps to https://www.pulumi.com/blog/pulumi-kubernetes-new-2022/
link CRDs to https://www.pulumi.com/blog/new-kubernetes-superpowers/#strongly-typed-kubernetes-crds

File: themes/default/content/product/_index.md [hunk 10]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
 
-        - description: Manage cloud applications with visibility and controls
+              - title: 
+                icon: monitor
+                icon_color: purple
+                description: |
+                    Find 100+ providers and cloud components in the Pulumi Registry. Bridge any existing Terraform provider to Pulumi.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:82]: link Pulumi Registry to the registry
link "bridge" to https://github.com/pulumi/pulumi-tf-provider-boilerplate

File: themes/default/content/product/_index.md [hunk 11]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
+        - description: Not just a CLI, but Infrastructure as Software.
           features:
-              - title: Policy as Code
+              - title: Pulumi CLI (open source)
                 icon: shield
                 icon_color: purple
                 description: |
-                    [Enforce compliance and detect drift](/crossguard/) by checking infrastructure against rules for
-                    security, cost, and best practices.
+                    Run IaC actions from your workstation with the open source command line interface.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:90]: link the CLI to: https://github.com/pulumi/pulumi

File: themes/default/content/product/_index.md [hunk 12]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
 
-              - title: State and secrets management
+              - title: Pulumi as a Library (open source)
                 icon: security
                 icon_color: purple
                 description: |
-                    [Pulumi Service](/product/pulumi-service/) manages your infrastructure state and automatically
-                    encrypts sensitive values in transit and at rest.
+                    Program IaC actions into your application code with the open source Automation API library.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:96]: link Automation API library to https://www.pulumi.com/docs/guides/automation-api/

File: themes/default/content/product/_index.md [hunk 13]
```diff
@@ -12,80 +12,115 @@ subheading: |
...
 
-              - title: Administer teams and stacks
+              - title: Pulumi as a REST API (service)
                 icon: monitor
                 icon_color: purple
                 description: |
-                    Secure access to infrastructure with role-based permissions and [single sign-on](/docs/guides/saml/). View deployed resources, review audit logs, and set tags.
+                    Build IaC actions into applications or workflows with the Pulumi Deployments API service.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:102]: link Pulumi Deployments to https://www.pulumi.com/product/pulumi-deployments/

File: themes/default/layouts/page/product.html [hunk 14]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
-        <div class="shape-background product-stats-section">
+    <section class="container mx-auto text-center relative">
+        <div class="mx-auto card bg-white w-full my-4 lg:my-0 text-center p-6 lg:p-8 mx-0 lg:mx-4 z-10 flex flex-row items-center">
+            <div class="w-1/3">
+                <h4>Build infrastructure automation into your software and applications</h4>
+            </div>
+            <div class="w-1/3"></div>
+            <div class="flex flex-col bg-white w-full lg:w-1/4 my-4 lg:my-0 text-center p-6 mx-0 z-10">
+                <img class="h-12" src="/images/product/altana.png" alt="Altana logo" />
+                <p>Built a custom CLI for its developers that enables them to easily deploy cloud architectures to AWS Used Pulumi’s Automation API library.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:133]:
```suggestion
                <p>Built a custom CLI for its developers that enables them to easily deploy cloud architectures to AWS using Pulumi’s Automation API library.</p>
```

File: themes/default/layouts/page/product.html [hunk 15]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+                <img class="h-12" src="/logos/customers/elkjop-nordic-logo.svg" alt="Altana logo" />
+                <p>Built an infrastructure provisioning application that enables developers to self-service dev environments on Azure</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="mt-12">
+        <div class="container mx-auto text-center w-2/3">
+            <h2>Deliver infrastructure through your software delivery pipeline</h2>
+            <p class="text-lg">Version, review, test, and deploy infrastructure code using the same tools and pipelines used for application code.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:149]:
```suggestion
            <p class="text-lg">Version, review, test, and deploy infrastructure code using the same tools used for application code.</p>
```

File: themes/default/layouts/page/product.html [hunk 16]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+                    <img class="h-12 px-5" src="/logos/tech/ci-cd/gitlab-ci.svg" alt="GitLab logo" />
+                    <img class="h-12 px-5" src="/logos/tech/ci-cd/azure-devops.svg" alt="Azure devops logo" />
+                    <img class="h-12 px-5" src="/logos/tech/ci-cd/aws-codedeploy.svg" alt="AWS code deploy logo" />
+                    <!-- find bitbucket logo -->
+                </div>
+            </div>
+
+            <div class="w-2/3 text-center">
+                <img src="/images/product/code-review.png" alt="Code review" />
+                <p>GitHub is shown above. Pulumi integrates with GitHub repositories and GitHub actions.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:174]:
```suggestion
                <p>GitHub is shown above. Pulumi integrates with GitHub repositories and GitHub Actions.</p>
```

File: themes/default/layouts/page/product.html [hunk 17]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+        </div>
+
+        <div class="container mx-auto text-center">
+            <div class="my-16 lg:my-24">
+                <h3>Test, validate, and shift left</h3>
+                <div class="flex flex-col lg:flex-row">
+                    <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                        {{ partial "color-icon.html" (dict "icon" "shield" "icon_color" "purple") }}
+                        <h5>Unit testing</h5>
+                        <p class="key-feature-description">Run fast, in-memory tests that mock resource calls with your preferred test frameworks.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:194]: link to https://www.pulumi.com/docs/guides/testing/unit/

File: themes/default/layouts/page/product.html [hunk 18]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+                    </div>
+                    <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                        {{ partial "color-icon.html" (dict "icon" "security" "icon_color" "purple") }}
+                        <h5>Integration testing</h5>
+                        <p class="key-feature-description">Run external black-box tests against ephemeral, running infrastructure.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:199]: link to https://www.pulumi.com/docs/guides/testing/integration/

File: themes/default/layouts/page/product.html [hunk 19]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+                    </div>
+                    <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                        {{ partial "color-icon.html" (dict "icon" "monitor" "icon_color" "purple") }}
+                        <h5>Policy as Code testing</h5>
+                        <p class="key-feature-description">Create and run resource-level assertions using policies written in general-purpose languages.</p>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:204]: link to https://www.pulumi.com/docs/guides/crossguard/

File: themes/default/layouts/page/product.html [hunk 20]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
+                    <li>Unified infrastructure visibility and search</li>
+                    <li>Collaborative workflows, notifications, and automations</li>
+                    <li>Dashboards & activity timelines</li>
+                    <li>Organize and visualize projects and stacks</li>
+                    <li>Git & CI/CD integrations</li>
+                    <li>Hosted deployments</li>
+                    <li>Programmatic controls and APIs</li>
+                    <li>Policy as Code enforcement</li>
+                    <li>Audit history and logging</li>
+                    <li>SaaS or Self-hosted</li>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:250]: link self-hosted to https://www.pulumi.com/product/self-hosted/

File: themes/default/layouts/page/product.html [hunk 21]
```diff
@@ -111,46 +122,169 @@ <h5>{{ $feature.title }}</h5>
...
                 </div>
             </div>
         </div>
+    </section>
+
+    <section class="container mx-auto text-center">
+        <h3 class="mb-10">Continuous Delivery</h3>
+        <div class="mx-auto card bg-white w-full my-4 lg:my-0 text-center p-6 lg:p-8 mx-0 lg:mx-4 z-10 flex flex-row">
+            <div class="w-1/4">
+                <h5>Integrate your CI/CD pipeline to automate infrastructure delivery</h5>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:215]: link ci/cd pipeline to https://www.pulumi.com/docs/guides/continuous-delivery/

**Comment by sean1588** [comment]
@gthuang I've added the images and the links you suggested. I also made a few more adjustments to improve the spacing. Hopefully I didn't miss any of your comments, but let me know if I did. I believe I got to all of them. Let me know what you think. This should be pretty close now.

File: themes/default/content/product/_index.md [hunk 22]
```diff
@@ -12,80 +12,103 @@ subheading: |
...
+                img: /images/product/features/pulumi_registry.svg
+                description: |
+                    Find 100+ providers and cloud components in the [Pulumi Registry](https://pulumi.com/registry). [Bridge](https://github.com/pulumi/pulumi-tf-provider-boilerplate) any existing Terraform provider to Pulumi.
+
+        - description: Not just a CLI, but Infrastructure as Software.
           features:
-              - title: Policy as Code
-                icon: shield
-                icon_color: purple
+              - title: Pulumi CLI (open source)
```
Comment (gthuang) [inline themes/default/content/product/_index.md:77]: the titles above the 3 icons in this section are not showing up in the preview

File: themes/default/content/product/_index.md [hunk 23]
```diff
@@ -12,80 +12,103 @@ subheading: |
...
     title: Key features
     items:
-        - description: Build [Infrastructure as Code](/what-is/what-is-infrastructure-as-code/) in familiar languages
+        - description:
           features:
-              - title: Languages you love
-                icon: code
-                icon_color: yellow
+              - title:
+                img: /images/product/features/pulumi_languages.png
```
Comment (gthuang) [inline themes/default/content/product/_index.md:27]: the image looks a little stretched out here

File: themes/default/content/product/_index.md [hunk 24]
```diff
@@ -12,80 +12,103 @@ subheading: |
...
                 description: |
-                    Use TypeScript, JavaScript, Python, Go, .NET, Java, and YAML to model cloud infrastructure
-                    by leveraging the features of each language.
+                    [Declarative IaC](https://pulumi.com/docs/intro/concepts/how-pulumi-works/) in TypeScript/JavaScript, Python, Go, C#, Java, and YAML. Deep runtime integrations and no proprietary frameworks.
 
-              - title: Build on any cloud
-                icon: global
-                icon_color: yellow
+              - title:
+                img: /images/product/features/pulumi_ides.png
```
Comment (gthuang) [inline themes/default/content/product/_index.md:32]: the image looks a little stretched out here

File: themes/default/content/product/_index.md [hunk 25]
```diff
@@ -12,80 +12,103 @@ subheading: |
...
+                    Run rapid [unit tests](https://www.pulumi.com/docs/guides/testing/unit/) with any supported framework to validate your infrastructure code before deployment.
 
-        - description: Deploy cloud infrastructure and applications together
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
-                icon: rocketship
-                icon_color: salmon
+              - title:
+                img: /images/product/features/pulumi_reusability.png
```
Comment (gthuang) [inline themes/default/content/product/_index.md:44]: the image looks a little stretched out here

File: themes/default/layouts/page/product.html [hunk 26]
```diff
@@ -111,53 +122,185 @@ <h5>{{ $feature.title }}</h5>
...
+                            >Create and run resource-level assertions using policies written in general-purpose languages.</a
+                        >
+                    </div>
                 </div>
             </div>
         </div>
+    </section>
+
+    <section class="container mx-auto text-center">
+        <h3 class="mb-10">Continuous Delivery</h3>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:221]: the box with the drop shadow surrounding this section isn't necessary

File: themes/default/content/product/_index.md [hunk 27]
```diff
@@ -12,80 +12,103 @@ subheading: |
...
+                description: |
+                    Use a single workflow to manage cloud native infrastructure like [Kubernetes](https://www.pulumi.com/registry/packages/kubernetes/), [Helm](https://www.pulumi.com/blog/helm-release-resource-for-kubernetes-generally-available/), [CRDs](https://www.pulumi.com/blog/new-kubernetes-superpowers/#strongly-typed-kubernetes-crds), [GitOps](https://www.pulumi.com/blog/pulumi-kubernetes-new-2022/), and more.
 
-        - description: Manage cloud applications with visibility and controls
+              - title: 
+                img: /images/product/features/pulumi_registry.svg
+                description: |
+                    Find 100+ providers and cloud components in the [Pulumi Registry](https://pulumi.com/registry). [Bridge](https://github.com/pulumi/pulumi-tf-provider-boilerplate) any existing Terraform provider to Pulumi.
+
+        - description: Not just a CLI, but Infrastructure as Software.
```
Comment (gthuang) [inline themes/default/content/product/_index.md:75]:
```suggestion
        - description: Not just a CLI, but Infrastructure as Software
```

File: themes/default/layouts/page/product.html [hunk 28]
```diff
@@ -111,53 +122,185 @@ <h5>{{ $feature.title }}</h5>
         </div>
     </section>
 
-    <section id="stats" class="text-center lg:mt-32 lg:pb-32 relative">
-        <div class="shape-background product-stats-section">
+    <section class="container mx-auto text-center relative">
+        <div class="mx-auto card bg-white w-full my-4 lg:my-0 text-center p-6 lg:p-8 mx-0 lg:mx-4 z-10 flex flex-row items-center">
+            <div class="w-1/2 pr-20">
+                <h4>Build infrastructure automation into your software and applications</h4>
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:128]:
```suggestion
                <h4>Build infrastructure automation into your software and applications with Pulumi Automation API</h4>
```

File: themes/default/layouts/page/product.html [hunk 29]
```diff
@@ -111,53 +122,185 @@ <h5>{{ $feature.title }}</h5>
...
+            </div>
+            <div class="w-full flex flex-row">
+                <div class="flex flex-col bg-white w-full my-4 lg:my-0 text-center p-6 mx-0 z-10">
+                    <img class="h-12" src="/images/product/altana.png" alt="Altana logo" />
```
Comment (gthuang) [inline themes/default/layouts/page/product.html:132]: is there a way to make this logo smaller? it looks much bigger than the other two next to it

**Comment by aaronkao** [comment]
It is really hard to tell what are the "big" sections in the page. There are a ton of subsections. So I suspect the plan of this page was originally the call out each of the major blocks of the diagram correct? 

I would have expected there to be these sections:  IaC Engine, Deploy to Any Cloud, Consume and Author in Any Language, Infrastructure Delivery Pipeline, Pulumi Service, Create Orchestration Workflows. 

The goal is to explain the theory or strategy of the marketecture. Otherwise the ordering and organization is too chaotic. The current page goes from three columns, two columns, single columns, horizontal blades. The UI isn't conducive to understanding the page intent.

**Comment by gthuang** [comment]
> It is really hard to tell what are the "big" sections in the page. There are a ton of subsections. So I suspect the plan of this page was originally the call out each of the major blocks of the diagram correct?
> 
> I would have expected there to be these sections: IaC Engine, Deploy to Any Cloud, Consume and Author in Any Language, Infrastructure Delivery Pipeline, Pulumi Service, Create Orchestration Workflows.
> 
> The goal is to explain the theory or strategy of the marketecture. Otherwise the ordering and organization is too chaotic. The current page goes from three columns, two columns, single columns, horizontal blades. The UI isn't conducive to understanding the page intent.

the page has each of the sections that you called out. the section that's titled "Not just a CLI, but Infrastructure as Software" could be renamed to "Create Orchestration Workflows." could you clarify what you mean by your comment? i'm assuming it's something about the spacing or font sizing?

**Comment by sean1588** [comment]
going to close this out for now since we decided we will revisit the overall design of this page after the new year.

**Pull request closed without merging**